app/lib/proxy_api/remote_execution_ssh.rb

```diff
@@ -11,6 +11,12 @@ def pubkey
       raise ProxyException.new(url, e, N_('Unable to fetch public key'))
     end
 
+    def ca_pubkey
+      get('ca_pubkey')&.strip
+    rescue
+      Rails.logger.info("Unable to fetch CA public key. Using public key authentication instead.")
```

Comment:
So if the proxy doesn't have the ca_pubkey endpoint, the get will raise and we'll get this warning. If the proxy has that endpoint, but doesn't have a ca_pubkey configured, the get will silently return an empty string, right?

Reply:
Right. Looking at it now I don't know why I made it like this, the message makes little sense. I updated it to reflect the behavior of the pubkey api call.